CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 <a name="unreleased"></a>
 ## Unreleased
 
+<a name="v2.15.5"></a>
+## v2.15.5 (2018-06-06)
+
+- API version 2.13
```

Comment:
You don't need to fill these out, I typically do before the release. It will likely be a minor bump anyway due to subscription terms feature.